### ISSUE TITLE
Broken i18next v3 plural for spanish

### ISSUE THREAD

**Issue by SamantazFox** [translate/translate#4873]
Seems like a similar issue to https://github.com/translate/translate/issues/4812 is back, but only for spanish:
https://github.com/iv-org/invidious/pull/3708/commits/e8063db98b6d16710544115ebf1f96cc2a231e94

**Comment by nijel** [comment]
https://github.com/translate/translate/issues/4812 was not really fixed. The issue is merely on i18next side, but their approach to address this is to use v4 format. Can you switch to that?

**Comment by SamantazFox** [comment]
> Can you switch to that?

Unfortunately, not at the moment, due to a lack of time.
For the record, we use i18next plural format, but we don't actually use i18next, as our project is written in Crystal.

**Comment by SamantazFox** [comment]
I updated the title, because it impacts more languages than previously reported.
Could it be a weblate-only bug now?

**Comment by nijel** [comment]
No, the issue needs to be addressed in both translate-toolkit and Weblate:

- translate-toolkit needs to know non-standard plurals i18next v3 uses
- Weblate needs definitions for these rules and apply them with this format

PS: Can you please update the initial post so that it is clear what is going on? The referenced commit no longer exist...

**Comment by SamantazFox** [comment]
Done!
